_layouts/frontpage.html

```diff
@@ -384,6 +384,12 @@ <h2>Brought to you by</h2>
               -->
             </div>
             <div class="col-md-12 text-center">
+              <span class="subtitle col-sm-12">Community representatives:</span>
+              <div class="text-center back-level clearfix">
+                <a href="https://github.com/bvenners" class="col-md-4 col-xs-12" target="_blank">Bill Venners</a>
+                <a href="https://github.com/tpolecat" class="col-md-4 col-xs-12" target="_blank">Rob Norris</a>
```

Comment:
```suggestion
                <a href="https://github.com/tpolecat" class="col-md-6 col-xs-12" target="_blank">Rob Norris</a>
```

Reply:
This is how it looks like with those changes:

![Screenshot from 2020-01-29 09-32-25](https://user-images.githubusercontent.com/332812/73340073-548a1400-427a-11ea-8f4a-dd7da7576c1a.png)